### PR TITLE
add sections pages and link them to the main page

### PR DESCRIPTION
**Pull request by tahmeedshah**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,8 @@
     "lodash": "^4.17.21",
     "luxon": "^3.2.1",
     "require-css": "^0.1.10",
-    "vue": "^3.4.15"
+    "vue": "^3.4.15",
+    "vue-router": "^4.3.0"
```
Comment (sbilodeau) [inline package.json]: I think is `vue-router` must be a peer dependancies. what do you think @blaisefonseca

**Review by sbilodeau — COMMENTED** [review]
@blaisefonseca did we said we call files `preview.vue` or `index.vue`?

File: package.json (hunk 1 above)
Comment (blaisefonseca) [reply package.json]: @sbilodeau  I don't think we even need it. @tahmeedshah  we should use tab and load the main vue file from the folder. there is no need for router here.

**Comment by blaisefonseca** [comment]
> @blaisefonseca did we said we call files `preview.vue` or `index.vue`?

preview.vue, we will have to exclude these files in rollup later

File: package.json (hunk 1 above)
Comment (tahmeedshah) [reply package.json]: Removed the router and added tabs to load preview pages for each section.

**Pull request merged**